packages/frontend/web-ui/src/scss/home/_page.scss

```diff
@@ -33,3 +33,78 @@
     text-align: center;
   }
 }
+
+.home-auth-container {
+  padding: 2rem;
+}
+
+.home-auth-container .home-auth-text {
+  color: var(--cg-bkg-primary-dark);
+  padding: 2rem 0;
+  text-align: left;
+  font-size: variables.$font-md;
+}
+
+.home-auth-container .home-auth-text-white {
+  color: var(--cg-text-primary);
+  padding: 2rem 1rem;
+  text-align: left;
+  font-size: variables.$font-sm;
+}
+
+.home-auth-button-container {
+  display: flex;
+  justify-content: end;
+}
+
+.home-auth-button-container .home-auth-button-new-game {
+  background-color: var(--cg-bkg-primary-dark);
+  margin: 2rem auto;
+  padding: 1rem;
+  border-radius: 10px;
+  font-weight: bold;
+  box-shadow:
+    0 4px 8px 0 rgba(68, 2, 131, 0.5),
+    0 6px 20px 0 rgba(68, 2, 131, 0.5);
```

Comment:
This is probably duplicated, extract it to a different class

Reply:
Changed!